src/content/peerconnection/munge-sdp/js/main.js

```diff
@@ -58,31 +58,30 @@ getSources();
 
 function getSources() {
   if (typeof MediaStreamTrack === 'undefined') {
-    alert(
-      'This browser does not support MediaStreamTrack.\n\nTry Chrome Canary.');
+    alert('This browser does not support MediaStreamTrack.\n\nTry Chrome Canary.');
```

Comment:
I think this is left over from checking for MediaStreamTrack.getSources which got replaced by enumerateDevices ages ago. I'd just remove the whole function and call enumerateDevices directly.